### PR TITLE
JUnit Tests for Watchlist class

### LINKED ISSUE #33: Implement JUnit Test for Watchlist class

### PR DESCRIPTION
**Pull request by OngDeZhi**
Adds: 
1. WatchlistTest.class

Closes #33

### CODE REVIEW

**Review by EyoWeiChin — APPROVED** [review]
LGTM!

**Review by ChanJianHao — APPROVED** [review]
LGTM!

File: src/test/java/seedu/duke/WatchlistTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    }
+
+    @Test
+    void toString_EmptyAnimeList_printWatchlistNameAndListIsEmptyMessage() {
+        Watchlist watchlist = new Watchlist("myWatchlist");
+        String expected = "myWatchlist" + System.lineSeparator();
+        expected += "Uhh.. It's empty.. :(" + System.lineSeparator();;
+        assertEquals(expected, watchlist.toString());
+    }
+}
```
Comment (ChanJianHao) [inline src/test/java/seedu/duke/WatchlistTest.java]: Would be good if an extra line is added after EOF.

Comment (OngDeZhi) [reply src/test/java/seedu/duke/WatchlistTest.java]: Noted on that, I will add an extra line.

**Review by ChanJianHao — APPROVED** [review]
LGTM!

File: src/test/java/seedu/duke/WatchlistTest.java (hunk 1 above)
Comment (ChanJianHao) [reply src/test/java/seedu/duke/WatchlistTest.java]: All good now!

**Pull request merged**